src/controllers/apps/todo/todo.controllers.js

```diff
@@ -100,7 +100,7 @@ const toggleTodoDoneStatus = asyncHandler(async (req, res) => {
       new ApiResponse(
         200,
         todo,
-        "Todo marked " + todo.isComplete ? "done" : "undone"
+        "Todo marked " + `${todo.isComplete ? "done" : "undone"}`
```

Comment:
The original code has an issue with string concatenation logic. The condition should be evaluated correctly within the string interpolation.
Improvement: The use of string interpolation directly inside the template literal ensures clarity and correctness.